src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,90 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final byte EXPENSION_COEF = 2;
+    private int size;
+    private Node<K, V>[] table;
 
-    @Override
-    public void put(K key, V value) {
+    public MyHashMap() {
+        table = new Node[DEFAULT_INITIAL_CAPACITY];
+    }
 
+    public void put(K key, V value) {
+        if (size >= table.length * LOAD_FACTOR) {
+            resize();
+        }
```

Comment:
```suggestion
      resize();
```

Reply:
i dont uderstand what do you wanna I fix)

Reply:
if resize is only ever called inside of this `if`, we should put if inside of resize